api/src/prescription/learner-management/infrastructure/repositories/organization-learner-repository.js

```diff
@@ -73,10 +73,45 @@ const disableCommonOrganizationLearnersFromOrganizationId = function (organizati
   return knex('organization-learners').where({ organizationId }).update({ isDisabled: true, updatedAt: new Date() });
 };
 
+/**
+ * @function
+ * @name reconcileCommonOrganizationLearners
+ * @param {Object} params
+ * @param {number} params.organizationId
+ * @param {number} params.userId
+ * @param {Object} params.reconciliationInformations
+ * @returns {Promise<boolean>}
+ */
+const reconcileCommonOrganizationLearners = async function ({ organizationId, userId, reconciliationInformations }) {
```

Comment:
**To do :** cette méthode devrait simplement s'appeler "Update" et prendre des objets du domaine en entrée. Elle ne retournerait qu'un d'organizationLearner avec son userId. Cela permettrait de retirer la partie métier du repository

Reply:
on a opté pour un découpage find + update